### PR TITLE
firebase-dsym-upload 0.0.1

### PR DESCRIPTION
**Pull request by andrewmarmion**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/firebase-dsym-upload/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
```
Comment (trapacska) [inline steps/firebase-dsym-upload/0.0.1/step.yml]: please remove unused dependencies

File: steps/firebase-dsym-upload/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- fdu_fabric_location: null
```
Comment (trapacska) [inline steps/firebase-dsym-upload/0.0.1/step.yml]: please include the suggested value as a default input:
```
- fdu_fabric_location: "./Pods/Fabric/upload-symbols"
```

File: steps/firebase-dsym-upload/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    summary: Summary. No more than 2-3 sentences.
+    title: Location of Fabric
+- fdu_google_services_location: null
+  opts:
+    description: |
+      The locations is usually in the form ./YOUR-APP-NAME/GoogleService-Info.plist
+    is_expand: true
+    is_required: true
+    title: Location of your GoogleService-Info.plist
+- fdu_dsym_location: null
```
Comment (trapacska) [inline steps/firebase-dsym-upload/0.0.1/step.yml]: please include the suggested value as a default input:
```
- fdu_dsym_location: "$BITRISE_DSYM_PATH"
```

File: steps/firebase-dsym-upload/0.0.1/step.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- fdu_fabric_location: null
+  opts:
+    description: "This is the location of where Fabric is stored. \nIt is usually
+      in ./Pods/Fabric/upload-symbols\n"
```
Comment (trapacska) [inline steps/firebase-dsym-upload/0.0.1/step.yml]: so you no need to include suggested input value in the description of the input

File: steps/firebase-dsym-upload/0.0.1/step.yml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  opts:
+    description: |
+      The locations is usually in the form ./YOUR-APP-NAME/GoogleService-Info.plist
+    is_expand: true
+    is_required: true
+    title: Location of your GoogleService-Info.plist
+- fdu_dsym_location: null
+  opts:
+    description: |
+      You should use $BITRISE_DSYM_PATH here
```
Comment (trapacska) [inline steps/firebase-dsym-upload/0.0.1/step.yml]: so you no need to include suggested input value in the description of the input

**Comment by andrewmarmion** [comment]
I will perform the updated changes. Thank you for taking the time to review my PR.

**Pull request closed without merging**